gptme/message.py

```diff
@@ -52,6 +52,11 @@ class Message:
 
     def __post_init__(self):
         assert isinstance(self.timestamp, datetime)
+        assert (
+            self.content.rstrip() == self.content
+        ), "Content should not have trailing whitespace: {}".format(
+            "\n".join(self.content.splitlines()[-3:])
+        )
```

Comment:
This will break things like reading conversations from before this requirement. Should make the tests pass with it and then comment it out, perhaps only keep it enabled in tests.